src/main/java/seedu/address/ui/PersonDetailsPanel.java

```diff
@@ -0,0 +1,90 @@
+package seedu.address.ui;
+
+import java.util.Comparator;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.Region;
+import seedu.address.model.person.Person;
+import seedu.address.model.tag.Tag;
+
+/**
+ * A UI component that displays the details of a {@code Person}.
+ */
+public class PersonDetailsPanel extends UiPart<Region> {
+    public static final String FXML = "PersonDetailsPanel.fxml";
+
+    @FXML
+    private Label name;
+    @FXML
+    private Label id;
+    @FXML
+    private Label phone;
+    @FXML
+    private Label address;
+    @FXML
+    private Label email;
+    @FXML
+    private FlowPane tags;
+    @FXML
+    private ImageView qrcode;
+    @FXML
+    private Label note;
+
+    /**
+     * Creates a new PersonDetailsPanel and clears all fields.
+     * Labels in the {@code .FXML} file contain text with the field name for convenience.
+     */
+    public PersonDetailsPanel() {
+        super(FXML);
+        clear();
+    }
+
+    /**
+     * Set fields with information from the person
```

Comment:
```suggestion
     * Set fields with information from the person.
```
Just a small nit (missing period) for JavaDoc comments!